docs/concepts/Autocompletion.md

```diff
@@ -0,0 +1,35 @@
+# Autocompletion
+
+## Prerequisites 
+
+Autocompletion currently works for the following shells: bash, zsh and fish.
+
+For bash, please install bash-completion.
+
+```
+brew install bash-completion
+```
+
+And the then add the following line to ~/.bash_profile or ~/.bashrc:
+
+```
+[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
+```
```

Comment:
1) based on omelette's suggestion https://github.com/f/omelette#manual-installation , should we include instruction for the other two? 

2) should we just tell users to run `echo '[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"' >> ~/.bash_profile` instead of describing what they need to do?

this means we clearly tell them you need to run two steps : 1) install bash-completion 2) run the script below. Tha't the easiest thing for developers

Reply:
1) It is related to manual installation. We use automatic installation with 'ask autocomplete setup' command.
2) I will update.

Reply:
done